include/unistd.h

```diff
@@ -259,7 +259,9 @@
 
 /* Helpers and legacy compatibility definitions */
 
+#ifndef CONFIG_LIBCXX
 #define link(p1, p2)                     symlink((p1), (p2))
```

Comment:
let's change link to normal function unconditionally...

Reply:
Done.